vscode/.prettierrc.mjs

```diff
@@ -0,0 +1,15 @@
+/** @type {import("prettier").Config} */
+const config = {
+  // default value used
+  // trailingComma: "all",
+  // semi: true,
+  // singleQuote: false,
+
+  // Values used from .editorconfig:
+  //   - printWidth == max_line_length
+  //   - tabWidth == indent_size
+  //   - useTabs == indent_style
+  //   - endOfLine == end_of_line
```

Comment:
Also need `.editorconfig`.  Copying from tackle2-ui would work.

Except ..... let's please have a wider `max_line_length`!  100 would be much nicer than 80.